f3/granite.go

```diff
@@ -665,22 +702,28 @@ func (q *quorumState) HasAgreement() bool {
 }
 
 // Checks whether at least one message has been received from a strong quorum of senders.
-func (q *quorumState) ReceivedFromQuorum() bool {
+func (q *quorumState) ReceivedFromStrongQuorum() bool {
 	return q.sendersTotalPower > q.powerTable.Total*2/3
 }
 
-// Checks whether a chain (head) has reached quorum.
-func (q *quorumState) HasQuorumAgreement(cid TipSetID) bool {
+// Checks whether a chain (head) has reached a strong quorum.
+func (q *quorumState) HasStrongQuorumAgreement(cid TipSetID) bool {
+	cp, ok := q.chainPower[cid]
+	return ok && cp.hasStrongQuorum
+}
+
+// Checks whether a chain (head) has reached weak quorum.
+func (q *quorumState) HasWeakQuorumAgreement(cid TipSetID) bool {
```

Comment:
This is now unused, but I guess we can leave it here for a while until we're sure it won't be needed.